docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -297,8 +298,53 @@ \section{Critical Assumptions}
 
 \section{Safety and Security Requirements}
 
-\wss{Newly discovered requirements.  These should also be added to the SRS.  (A
-	rationale design process how and why to fake it.)}
+\begin{enumerate}[label=SR\arabic*.]
+	\item The system shall protect personally identifiable information (PII) by encrypting information both
+	      at rest and in transit
+	\item[] \emph{Rationale:} There are legal consequences associated with losing people's PPI.
+	\item[] \emph{Associated Hazards:} H1-1 \\
+	\item The system shall allow the user to reset their password
+	\item[] \emph{Rationale:} If a user forgets their password, they will be unable to access information
+		regarding their work. Furthermore, losing access to their account due to a malicious actor would
+		allow unauthorized people access to components that require a level of authorization.
+	\item[] \emph{Associated Hazards:} H2-1, H2-2 \\
+	\item In the case of multiple appointments scheduled in the same time frame, the system shall allow the
+	      shop owner or employee to accept one of the appointments
```

Comment:
This requirement sounds like we will only ask the shop accept if there is a race condition.

Reply:
I'll rephrase it